evennia/web/utils/middleware.py

```diff
@@ -0,0 +1,60 @@
+from django.contrib.auth import authenticate, login
```

Comment:
I would think this belongs in the `evennia/web` folder since it's explicitly dealing with web-related stuff. Maybe in `web/utils` ?

Reply:
I'm not sure what you're suggesting? It already is in `evennia/web`, specifically in `web/utils`. We already had `backends` and `general context` there so this location felt appropriate.

Reply:
Oh, I'm apparently tired and blind. :man_facepalming: For some reason I read `utils` and immediately assumed it was `evennia/utils`. Never mind then, carry on.